kolibri/core/auth/test/test_api.py

```diff
@@ -359,9 +359,9 @@ def test_facility_user_can_get_detail(self):
             reverse("kolibri:core:facility-detail", kwargs={"pk": self.facility1.pk}),
             format="json",
         )
-        # .assertDictContainsSubset checks that the first argument is a subset of the second argument
-        self.assertDictContainsSubset(
-            {"name": self.facility1.name}, dict(response.data)
+        self.assertEqual(
+            dict(response.data),
+            {**dict(response.data), **{"name": self.facility1.name}},
```

Comment:
Note this is the line that is giving Python 2 syntax issues. Can fix by doing:
```suggestion
            dict(name=self.facility1.name, **dict(response.data)),
```